src/sample-executors/Multicall.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.16;
+
+/// @title Multicall
+/// @notice Code from https://github.com/Uniswap/v3-periphery/blob/main/contracts/base/Multicall.sol
+/// @notice Enables calling multiple methods in a single call to the contract
+abstract contract Multicall {
```

Comment:
Can you just import this as an interface or external contract?

Reply:
its from v3-periphery but that solidity version is different that this repo so can't directly import afaik